### ISSUE TITLE
Can't parse block #405040

### ISSUE THREAD

**Issue by Polycarpik** [OmniLayer/omnicore#357]
We are currently stuck with the problem that we can't parse block # 405040 because it is not valid JSON.
http://omnichest.info/requeststat.aspx?stat=getblocktx&block=405040

It contains field 

```
"data":"This is class "OMNI" of FREE TERRITORIES stock of which in this class there is 500,000 of a total authorized amount of 100,000,000 (One Hundred Million). Review the Certificate of Formation for the FREE TERRITORIES at http://certificate.freeterritories.ch"
```

Because of `"OMNI"` JSON is invalid.
Could you please do something with it?

**Comment by dexX7** [comment]
Hmm.. yeah. It should be escaped.

@zathras-crypto: can you hot-fix this on chest's side?

**Comment by zathras-crypto** [comment]
Hey guys, thanks for the report and my apologies for any inconvenience caused :(

I've taken a look and this is actually an issue with 'Chest engine, so is going to take a little more time to fix (as I'll need to patch the engine and then reparse on a staging instance before switching them over).

> @zathras-crypto: can you hot-fix this on chest's side?

Done, but only for this specific case - I'll push through the engine fix and associated stuff in the coming days for more complete coverage.

Thanks
Z

**Comment by dexX7** [comment]
@zathras-crypto: is this issue resolved?

**Comment by zathras-crypto** [comment]
I thought so but nope.  I corrected the error post-parsing at the time and patched 'Chest engine but the next time one came through (a few days ago) the same issue was present so I obviously didn't fix it well enough.  @achamely gave me the heads up and I did the same hotfix again.

'Chest has "per function" handling of strings and they're currently not all exactly the same so I'm redoing the string handling to a single function so I can centralize things like escaping characters and doing SQL injection prevention etc.

Long story short - not fixed properly yet but will be soon.

**Comment by dexX7** [comment]
Looks fixed to me. :)
